src/main/java/org/entando/kubernetes/client/core/DefaultEntandoCoreClient.java

```diff
@@ -1,26 +1,8 @@
 package org.entando.kubernetes.client.core;
 
-import com.jayway.jsonpath.JsonPath;
-import java.nio.file.Paths;
-import java.util.List;
 import org.entando.kubernetes.exception.web.HttpException;
-import org.entando.kubernetes.model.bundle.descriptor.ContentTemplateDescriptor;
-import org.entando.kubernetes.model.bundle.descriptor.ContentTypeDescriptor;
-import org.entando.kubernetes.model.bundle.descriptor.FileDescriptor;
-import org.entando.kubernetes.model.bundle.descriptor.FragmentDescriptor;
-import org.entando.kubernetes.model.bundle.descriptor.LabelDescriptor;
-import org.entando.kubernetes.model.bundle.descriptor.PageDescriptor;
-import org.entando.kubernetes.model.bundle.descriptor.PageTemplateDescriptor;
-import org.entando.kubernetes.model.bundle.descriptor.WidgetDescriptor;
-import org.entando.kubernetes.model.digitalexchange.ComponentType;
-import org.entando.kubernetes.model.entandocore.EntandoCoreComponentUsage;
-import org.entando.kubernetes.model.entandocore.EntandoCoreContentModel;
-import org.entando.kubernetes.model.entandocore.EntandoCoreFile;
-import org.entando.kubernetes.model.entandocore.EntandoCoreFolder;
-import org.entando.kubernetes.model.entandocore.EntandoCoreFragment;
-import org.entando.kubernetes.model.entandocore.EntandoCorePage;
-import org.entando.kubernetes.model.entandocore.EntandoCorePageTemplate;
-import org.entando.kubernetes.model.entandocore.EntandoCoreWidget;
+import org.entando.kubernetes.model.bundle.descriptor.*;
+import org.entando.kubernetes.model.entandocore.*;
```

Comment:
These imports are out of Entando's code styles and will generate lot's of unnecessary changes in the next PRs.
Please add the intellij codestyle and organize imports on the whole project before creating a PR.

**Entando Code Style:**
https://github.com/entando/entando-code-style